### DISCUSSION TITLE
se paran las impresiones

### DISCUSSION THREAD

**Discussion by bombardeo38** [Jyers/Marlin#1544] [Q&A]
pongo a imprimir con version 2.0.0 y cuando quiere deja de imprimir pero con 1.3.5 b no hay ese erorr a mi maquina es una ender 3 v2

**Comment by rodolfopaulo** [comment]
This has already happened to me also, along with another example. The printing job was in the last 3 or 4 minutes, and when I decided to leave the Tune menu, the print stopped and the printer simply rebooted. 
One more issue: When started one fresh print, using the same Gcode from a previous print, the process stopped all together. The printer went to the main menu.
Thanks for the great job, Jyers!!!

**Reply by FinlessBob** [reply]
I would see this on the 1.3.5 firmware too sometimes.

**Comment by zorrito93** [comment]
hola buenas noches , use el fimware 2.0.0 use el deafaut , imprimia pero a la hora de change filament extruia pero no seguia la impresion se quedaba a la espera de otra orden al igual  que cuando hice pausa de impresion no reanudaba , solo se reanudaba si apagaba y tenia el powerloss activado.
que tengan una buena noche o buen dia  :)
